vars/gctsRollback.groovy

```diff
@@ -0,0 +1,11 @@
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/gctsRollback.yaml'
+
+void call(Map parameters = [:]) {
+        List credentials = [
+        [type: 'usernamePassword', id: 'abapCredentialsId', env: ['PIPER_username', 'PIPER_password']]
```

Comment:
as indicated above, please add the token credential, ideally use same name as for the other GitHub related steps

Reply:
will do